src/Kernel/Class.class.st

```diff
@@ -275,16 +275,25 @@ Class >> category: aString [
 	any previous categorization."
 
 	| newPackage |
-
 	self flag: #package. "Should be deprecated"
 	self category = aString ifTrue: [ ^ self ].
 
 	self basicCategory: aString asSymbol.
 
-	self flag: #package. "#toTagName: should be inlined here."
 	newPackage := self packageOrganizer ensurePackageMatching: self category.
-	newPackage moveClass: self toTag: (newPackage toTagName: self category).
 
+	"The category can be
```

Comment:
I imagine that this is only used by the tonel importer? I mean normally except on such case even the classbuilder should not use this API so we could package this will tonel?

Reply:
I will merge now and may be we could improve the comment later. 
Or push this method to tonel if my hypothesis is correct.

Reply:
This is almost not used anymore. I just need to remove 2-3 senders then I’ll deprecate it because external projects could still be using it